composer.json

```diff
@@ -6,26 +6,20 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.8",
-        "php": ">=7.2"
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9.1"
```

Comment:
We cannot allow a version lower than 8.9.17 as <= 8.9.17 had a vulnerability. See https://www.drupal.org/sa-core-2021-004